### PR TITLE
Remove testing data from distribution

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #10330
#### Summary of Changes
- remove the testing files from the core stable versions
#### Testing Instructions
- run the build
- see that the testing files are not there
- run the installation
- see that there is no testing sample files option.

Thanks @brianteeman

### CODE REVIEW

**Comment by brianteeman** [comment]
Not even a 10 minute PR :)

**Comment by zero-24** [comment]
writing test instructions was the longest part :smile:

**Comment by andrepereiradasilva** [comment]
shouldn't it be removed in com_admin too?

**Comment by mbabker** [comment]
... no?  The entire installation directory is removed when you install a site so it doesn't persist across versions unless you core hacked the install check.

File: build/build.php [hunk 1]
```diff
@@ -119,6 +119,10 @@
 	'phpunit.xml.dist',
 	'tests',
 	'travisci-phpunit.xml',
+	// Get the testing sample data out of stable versions
```
Comment (mbabker) [inline build/build.php]: Nit picking here.  "Remove the testing sample data from all packages".

**Comment by andrepereiradasilva** [comment]
"entire installation directory" should be "removed when you install a site"

But yes, makes sense

**Comment by zero-24** [comment]
@mbabker

>Nit picking here. "Remove the testing sample data from all packages".

fixed with: https://github.com/joomla/joomla-cms/pull/10364/commits/f709404b82339521526899cf496d0589165a522a

Thanks.

**Comment by zero-24** [comment]
Is this something the PLT need to decide on?

**Comment by zero-24** [comment]
... or just no interest any more?

**Comment by brianteeman** [comment]
I am definitely in favour of this

On 2 June 2016 at 12:06, zero-24 <notifications@github.com> wrote:

> ... or just no interest any more?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/10364#issuecomment-223261221>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABPH8cLFZeAcbfnw2bNFFWE58IbOVtgNks5qHrkygaJpZM4Iaazy>
> .
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e2a81244e81181f2671439586f27bbe76b958fe2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10364">issues.joomla.org/joomla-cms/10364</a>.</sub>

**Comment by zero-24** [comment]
Thanks :)

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on bb637097560d92f1590fe6f08033fc808725c1ee<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10364">issues.joomla.org/joomla-cms/10364</a>.</sub>

**Comment by pritalpatel** [comment]
I was trying to test it and tried to follow testing instructions. but didn't able to understand how exactly I can test this Pull request. Somehow I reached to `build` folder and found `build.php` and it needs to run from cli. But that doesn't work for me when I have downloaded https://codeload.github.com/zero-24/joomla-cms/zip/removetestingfromstable and tried to run `php build.php` from build folder. 

Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10364">issues.joomla.org/joomla-cms/10364</a>.</sub>

**Comment by zero-24** [comment]
@pritalpatel can you explain what don't work? I have just add 3 files to be excluded from the core installation the script should run normaly. Can you confirm that the issues come from the changes here? Are you able to user the build.php from core?

**Comment by zero-24** [comment]
@brianteeman i have just synced to staging. Nothing new to test ;)

**Comment by bertmert** [comment]
I have tested this item :white_check_mark: successfully on 7811a67b5066e817a51852971eacc0c2ad86bc9e<br /><br />> run build

Newbie comment:
Puuh. After some hours it ran finally ;-)
testing datas not present in full packages anymore<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10364">issues.joomla.org/joomla-cms/10364</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 7811a67b5066e817a51852971eacc0c2ad86bc9e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10364">issues.joomla.org/joomla-cms/10364</a>.</sub>

**Comment by brianteeman** [comment]
Rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10364">issues.joomla.org/joomla-cms/10364</a>.</sub>

**Pull request merged**